device-types/Netgate/8200-max.yaml

```diff
@@ -0,0 +1,56 @@
+---
+manufacturer: Netgate
+model: 8200-max
+slug: netgate-8200-max
+u_height: 1
+is_full_depth: false
+console-server-ports:
+  - name: usb1
+    type: usb-a
+  - name: usb2
+    type: usb-a
+console-ports:
+  - name: console-usb
+    type: usb-micro-b
+    description: |
+      Auto-detect console port: use RJ-45 -OR- micro-USB.
+      115200 baud rate, 8 data bits, no parity, 1 stop bit, no flow control
+      or XON/OFF
+  - name: console-rj45
+    type: rj-45
+    description: |
+      Auto-detect console port: use RJ-45 -OR- micro-USB.
+      115200 baud rate, 8 data bits, no parity, 1 stop bit, no flow control
+      or XON/OFF
+power-ports:
+  - name: psu1
+    type: iec-60320-c6
```

Comment:
The power type of the device itself is `dc-terminal`. We are not modelling the power brick here

Please adjust accordingly